lib/hammer_cli/ca_cert_fetcher.rb

```diff
@@ -1,39 +1,63 @@
+require 'hammer_cli/ca_cert_manager'
 module HammerCLI
   class CACertFetcher
-    def self.fetch_ca_cert(host)
-      CACertFetcher.new.fetch_ca_cert(host)
-    end
-
-    def fetch_ca_cert(host)
+    def fetch_ca_cert(service_uri, ca_path)
       begin
-        uri = URI.parse(host)
+        uri = URI.parse(service_uri)
+        raise URI::InvalidURIError.new(scheme_error(uri)) unless uri.scheme == 'https'
```

Comment:
It's probably worth checking `host.uri` here too (or in the cert manager?). When host can't be parsed it produces symlink to `~/.hammer/certs/.pem`.

```
You can display the certificate content with
  $ openssl x509 -text -in /root/.hammer/certs/.pem
```